examples/mender.conf.demo

```diff
@@ -1,7 +1,7 @@
 {
   "InventoryPollIntervalSeconds": 5,
   "RetryPollIntervalSeconds": 30,
-  "ServerURL": "https://hosted.mender.io/",
-  "TenantToken": "Paste your Hosted Mender token here",
+  "ServerURL": "https://docker.mender.io",
+  "ServerCertificate": "/usr/share/doc/mender-client/examples/demo.crt",
```

Comment:
Maybe we can talk about it when you get to the office, but I'm a little bit confused by this. If we have the setup command, why do we need this file at all? And if we do need it, should it not still point to our first class citizen, `hosted.mender.io`?

Reply:
(I am WFH today)

I don't really know what to do with this file. It is very much open for discussion.

It is still used in
- `Dockerfile.daemon`, where we want to have the demo certificate for a demo server.
- `mender-convert` (v1) where we want to have demo timing settings but `hosted.mender.io`.
- Also in the upstream Debian packages both conf files are installed as examples (when we unify recipes, we will do this also in our own deb package).

So, overall I thought it would be a good idea to keep the demo as both demo settings and demo server. The only broken piece with this decision would be mender-convert (v1).

Reply:
`Dockerfile.daemon` I would discount completely. Tests should not influence what examples we install.

IMHO I think Debian examples should also point to `hosted.mender.io`. It is, ultimately, where we want people to go, even if the slightly fake `docker.mender.io` is also an option. This is especially true because `docker.mender.io` won't work without additional configuration. So I think both files should point to Hosted.

If we really want to provide an example, I think we should make a separate one, `mender.conf.custom-server`, or something like that.

Reply:
I think that's ok.